docs/kpod-ps.1.md

```diff
@@ -108,6 +111,14 @@ sudo kpod ps -a --format "{{.ID}}  {{.Image}}  {{.Labels}}  {{.Mounts}}"
 69ed779d8ef9f  redis:alpine  batch=no,type=small  proc,tmpfs,devpts,shm,mqueue,sysfs,cgroup,/var/run/,/var/run/
 ```
 
+```
+sudo kpod ps --ns -a
+CONTAINER ID    NAMES                                                                   PID     CGROUP       IPC          MNT          NET          PID          USER         UTS
```

Comment:
I'm very confused by two "PID" columns.  Is this why you converted CtrPID to PID up above?  If so I think I'd keep it CtrPID above and not change it.

Reply:
I think one is the PID of the container and the other is the PID namespace of the container. This could probably be labeled better.

Reply:
Should label it as PIDNS to clarify.

Reply:
fixed.